tests/test_flow/test_persistence_compatability.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from .generate_test_compatability_cache import Harness
+
+
+@pytest.fixture
+def older_serialized_cache_harness(builder, datadir):
```

Comment:
So... at this point, is `datadir` doing anything except for copying the cached files into a temp directory? If not, it seems like it's only saving us about 1 line of code.

Reply:
Yes but I'm also using datadir in relative cache path test so it's already checked in with bionic now